setup.cfg

```diff
@@ -1,3 +1,16 @@
 [bdist_wheel]
 # code is written to work on both Python 2 and Python 3
-universal=1
+universal = 1
+
+[pep8]
+max-line-length = 119
+# Ignore E127: checked by pylint
+# E127 continuation line over-indented for visual indent
+ignore = E127
+exclude = migrations,setup.py
+
+[pydocstyle]
+match-dir = (?!tests|migrations|\.).*
+match = (?!setup).*.py
+add-select = D203,D404
```

Comment:
I'm slightly in favour of D211 since this is the new official PEP 257's stance and it compacts the files a bit.